### PR TITLE
[MODULAR] Converts table flipping feedback to balloon alerts

### PR DESCRIPTION
**Pull request by nikothedude**
## About The Pull Request

Title.

## How This Contributes To The Skyrat Roleplay Experience

Its easy to miss someone doing this, but it's a mechanics-heavy action, given how much of a defensive boon a flipped table is. It's useful to notice when someone is doing this.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
</details>

## Changelog

:cl:
qol: Table flipping feedback is now in the form of balloon alerts
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/tableflip/code/flipped_table.dm [hunk 1]
```diff
@@ -50,13 +50,13 @@
 	. = ..()
 	if(!istype(user) || !user.can_interact_with(src) || iscorticalborer(user))
 		return FALSE
-	user.visible_message(span_danger("[user] starts flipping [src]!"), span_notice("You start flipping over the [src]!"))
+	user.balloon_alert_to_viewers("righting table...")
```
Comment (vinylspiders) [inline modular_skyrat/modules/tableflip/code/flipped_table.dm]:
```suggestion
	user.balloon_alert_to_viewers("flipping table upright...")
```

File: modular_skyrat/modules/tableflip/code/flipped_table.dm [hunk 2]
```diff
@@ -50,13 +50,13 @@
...
 	if(do_after(user, max_integrity * 0.25))
 		var/obj/structure/table/new_table = new table_type(src.loc)
 		new_table.update_integrity(src.get_integrity())
 		if(custom_materials)
 			new_table.set_custom_materials(custom_materials)
-		user.visible_message(span_danger("[user] flips over the [src]!"), span_notice("You flip over the [src]!"))
+		user.balloon_alert_to_viewers("table righted")
```
Comment (vinylspiders) [inline modular_skyrat/modules/tableflip/code/flipped_table.dm]:
```suggestion
		user.balloon_alert_to_viewers("table flipped upright!")
```

File: modular_skyrat/modules/tableflip/code/flipped_table.dm [hunk 3]
```diff
@@ -89,16 +89,14 @@
 	flipped_table.set_custom_materials(custom_materials)
 
 	var/sound_volume = 100
-	var/visible_message = "[user] flips over the [src]!"
-	var/self_message = "You flip over the [src]!"
+	var/balloon_message = "table flipped"
```
Comment (vinylspiders) [inline modular_skyrat/modules/tableflip/code/flipped_table.dm:92]:
```suggestion
	var/balloon_message = "table flipped!"
```

**Review by vinylspiders — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/tableflip/code/flipped_table.dm (hunk 2 above)
Comment (nikothedude) [reply modular_skyrat/modules/tableflip/code/flipped_table.dm]: im pretty sure balloon alerts with an exclamation mark are generally reserved for failure alerts, ill ask

Comment (nikothedude) [reply modular_skyrat/modules/tableflip/code/flipped_table.dm]: yep, it is

**Review by vinylspiders — APPROVED** [review]

**Pull request merged**